src/bitdrift_public/protobuf/client/v1/api.proto

```diff
@@ -425,19 +431,19 @@ message ConfigurationUpdate {
     reserved 2;
 
     // The list of buffers.
-    bitdrift_public.protobuf.config.v1.BufferConfigList buffer_config_list = 3;
+    config.v1.BufferConfigList buffer_config_list = 3;
 
     // The workflows configuration.
-    bitdrift_public.protobuf.workflow.v1.WorkflowsConfiguration workflows_configuration = 4;
+    workflow.v1.WorkflowsConfiguration workflows_configuration = 4;
 
-    // The list of active bdtail sessions.
-    bitdrift_public.protobuf.bdtail.v1.BdTailConfigurations bdtail_configuration = 6;
+    reserved "bdtail_configuration";
+    reserved 6;
```

Comment:
I can continue to support old SDKs but I'm honestly not sure it's worth it given usage, and the fact that most usage is during development anyway. We can discuss continuing to support old clients or not.